source/adios2/operator/OperatorFactory.cpp

```diff
@@ -14,12 +14,8 @@
 #include "adios2/operator/plugin/PluginOperator.h"
 #include <numeric>
 
-#ifdef ADIOS2_HAVE_BLOSC
-#include "adios2/operator/compress/CompressBlosc.h"
-#endif
-
 #ifdef ADIOS2_HAVE_BLOSC2
-#include "adios2/operator/compress/CompressBlosc2.h"
+#include "adios2/operator/compress/CompressBlosc.h"
```

Comment:
Should we maybe be consistent and rename `adios2/operator/compress/CompressBlosc.h` to `adios2/operator/compress/CompressBlosc2.h` since the macros we use all use `2` as a suffix (`ADIOS2_HAVE_BLOSC2`)